Sources/Rendering/Core/RenderWindow/index.js

```diff
@@ -118,9 +134,14 @@ export function extend(publicAPI, model, initialValues = {}) {
 
   // Build VTK API
   macro.obj(publicAPI, model);
+
+  model.apiSpecificViews = [];
```

Comment:
```
const VIEW_CONSTRUCTORS = Object.create(null); // same as {}

publicAPI.registerViewConstructor = (name, constructor) => VIEW_CONSTRUCTORS[name] = constructor;
publicAPI.newAPISpecificView = (name, initialValues={}) => {
   return VIEW_CONSTRUCTORS[name || DEFAULT_API](initialValues);
}
```

Reply:
Also thinking that VIEW_CONSTRUCTORS and registerViewConstructor should be static.